doc/SCALING.md

```diff
@@ -0,0 +1,106 @@
+
+#### More on the graceful shutdown
+
+Note that the default TURN refresh lifetime is 10 minutes so STUNner may remain alive well after the last client goes away. This occurs when an UDP allocation is left open by a client (spontaneous UDP client-side connection closure cannot be reliably detected by the server). In such cases, after 10 mins the allocation will timeout and get deleted, which will then let `stunnerd` to terminate. 
+This feature enables the full support for graceful scale-down: the user can scale the number of `stunner` instances up and down as they wish and no harm should be made to active client connections meanwhile. 
+Caveats: 
+- currently the max lifetime for `stunner` to remain alive after deleted in 1 hour: this means that `stunner` will remain active only for 1 hour after it has been deleted/scaled-down. You can always set this in by adjusting the `terminationGracePeriod` on your `stunnerd` pods.
+- if there are active (or very recent) TURN allocations then the `stunner` pod may refuse to be removed after a kubectl delete. Use `kubectl delete pod --grace-period=0 --force -n stunner stunner-XXX` to force removal.
+
+
+## Example
+
+In this part we will walk you through a simple example on how to scale your `stunner` deployment. 
+
+A simple `HPA` config for scaling `stunner` would be:
+```yaml
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: hpa-stunner
+  namespace: stunner
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: stunner
+  minReplicas: 1
+  maxReplicas: 10
+  metrics:
+  - type: Resource
+    resource:
+      name: cpu
+      target:
+        type: Utilization
+        averageUtilization: 300
+```
+Configures resources in the STUNner deployment are the following.
+```yaml
+resources:
+  limits:
+    cpu: 2
+    memory: 512Mi
+  requests:
+    cpu: 500m
+    memory: 128Mi
+```
+
+Initially there is only a single `stunner` instance in the cluster. As new calls start to come in the amount of utilized CPU is increasing.  
+
+Scaling out is triggered when CPU usage of the `stunner` pod reaches 1500m (three times the requested CPU) core CPU.
+
+More calls come in and the summarized CPU usage by `stunner` pods reach 3000m core CPU, this is 1500m core on average. Scale out happens again. 
+
+As the the meeting is ended and participants leave the room the load will drop and the average CPU will fall under 3000m. Scaling in happens as the `HPA` removes one of the `stunner` instances. And so on...
```

Comment:
the the -> the

As participants levate the room at the end of the meeting, the load will drop ...